sugondat/chain/runtimes/test/src/lib.rs

```diff
@@ -21,6 +21,7 @@ use sp_runtime::{
     transaction_validity::{TransactionSource, TransactionValidity},
     ApplyExtrinsicResult, FixedPointNumber, Perquintill,
 };
+pub use pallet_transaction_payment::Multiplier;
```

Comment:
Why did you remove the `pub use` from the other line and add it in a new line?

Reply:
I wanted to re export only whats needed which is the Multiplier

Reply:
Yes but again we will define a new type alias multiplier for consistency

Reply:
We should re-export from primitives and use it here & in node rather than re-exporting from the runtimes.